game/models.py

```
@@ -99,6 +99,81 @@ def save(self, *args, **kwargs):
+    def __str__(self):
+        return (
+            f"{self.user.username} "
+            f"(Rating {self.rating})"
+        )
+
+class RatingHistory(models.Model):
+    RESULT_CHOICES = [
+        ("win", "Win"),
+        ("loss", "Loss"),
+        ("draw", "Draw"),
+    ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)